content/news/2023_12_19_annual_report.md

```diff
@@ -0,0 +1,162 @@
+  <img src="/images/news/GRASS_people_on_hilltop_2023.jpg"
+   alt="Annual Report for 2023"
+   title="GRASS GIS community"
+   width="45%" style="float:right;padding-left:25px;padding-top:15px">
+</a>
+
+2023 has been an amazing year in the advancement of the GRASS GIS project, which celebrated its 40th birthday!
+As the year draws to a close, let's look back at some of the achievements of the GRASS community.
+
+### Community Meeting
+
+The [GRASS GIS Community Meeting](/news/2023_08_13_grass_community_meeting_prague_june_2023_report) held at the Faculty of Civil Engineering of the Czech Technical University in Prague during the first days of June
+brought together users, supporters, contributors, power users and developers to celebrate, collaborate and chart the future of GRASS GIS.
+Thanks to the generous funding from OSGeo, FOSSGIS e.V., OSGeo.JP, and others, we were able to welcome 17 in-person participants from 9 countries on 3 continents and 4 remote participants!
+
+Moreover, in November, several members of the GRASS community were able to participate in-person and remotely in the [OSGeo Community Sprint](https://wiki.osgeo.org/wiki/OSGeo_Community_Sprint_2023) in Vienna!
+
+
+### Development Activity
+<a href="/images/news/grass83_news_screenshot.png">
+  <img src="/images/news/grass83_news_screenshot.png"
+   alt="GRASS GIS 8.3.0 screenshot"
+   title="GRASS GIS 8.3.0 screenshot"
+   width="35%" style="float:left;padding-right:25px;padding-top:15px">
+</a>
+
+In 2023, the maintainers published the [8.3.0](/news/2023_06_24_grass_gis_8_3_released/) feature release and three maintenance releases ([7.8.8](/news/2023_08_06_grass_gis_7_8_8_released/), [8.2.1](/news/2023_01_21_grass_gis_8_2_1_released/), and [8.3.1](/news/2023_10_25_grass_gis_8_3_1_released/)).
+The feature release brought, among other things, the single window layout in GUI by default,
+new support for parallelization in various raster modules, and several new options added to temporal modules.
+
+GRASS developer community was able to create 418 pull requests (PRs),
+merge 387 PRs and solve 103 issues. Furthermore, [9 new addons](https://github.com/OSGeo/grass-addons/pulls?q=is%3Aclosed+is%3Apr+label%3A%22new+addon%22+merged%3A%3E2023-01-01) were added by 7 contributors.
+
+Kudos to the GRASS GIS [top 5 code contributors](https://github.com/OSGeo/grass/graphs/contributors?from=2023-01-01&to=2023-12-19&type=c) (based on commits):
+Nicklas Larsson, Tomas Zigo, Markus Neteler, Vaclav Petras and Anna Petrasova.
+Thanks to the other 30 core developers and contributors that contributed PRs to the project!
+We are also excited to announce we welcomed a new core contributor, Edouard Choinière!
+
+<!-- In December, Linda Karlovska was awarded a [GRASS Student Grant](https://grasswiki.osgeo.org/wiki/GRASS_GIS_Student_Grants) to improve a new history browser panel in GRASS GUI. We are looking forward to the enhancements in 2024! -->
```

Comment:
Given all but Martin (who agreed to co-mentor this project) voted affirmatively, I am uncommenting this:
```suggestion
In December, Linda Karlovska was awarded a [GRASS Student Grant](https://grasswiki.osgeo.org/wiki/GRASS_GIS_Student_Grants) to improve a new history browser panel in GRASS GUI. We are looking forward to the enhancements in 2024!
```